xchange-huobi/src/main/java/org/knowm/xchange/huobi/service/HuobiTradeServiceRaw.java

```diff
@@ -78,7 +78,7 @@ String placeHuobiLimitOrder(LimitOrder limitOrder) throws IOException {
                 String.valueOf(
                     ((HuobiAccountServiceRaw) exchange.getAccountService())
                         .getAccounts()[0].getId()),
-                limitOrder.getOriginalAmount().setScale(4, BigDecimal.ROUND_DOWN).toString(),
+                limitOrder.getOriginalAmount().toString(),
```

Comment:
toString can return scientific notation if an exponent is needed
I doubt that  Huobis API can handle that

Reply:
The originalAmount parameter in the huobis API has different precision requirements for different currency pairs, so the precision of the originalAmount parameter should be provided by the caller.

Reply:
If you set the wrong precision, you cannot place an order.